scripts/routes/tabbar.ts

```diff
@@ -35,6 +35,23 @@ store.subscribe(() => {
         btbItemCart.badge.text = '10+';
     }
 });
+let btbItemFavorite = new TabBarItem();
```

Comment:
We should use const if the variable will not change :)

Reply:
DONE.